src/Option/Rotate.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Mezcalito ImgproxyBundle.
+ *
+ * (c) Mezcalito <dev@mezcalito.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Mezcalito\ImgproxyBundle\Option;
+
+class Rotate extends Option
+{
+    public function getParts(): array
+    {
+        $parts = [];
+
+        if (!empty($this->params['angle'])) {
+            $parts['angle'] = $this->params['angle'];
```

Comment:
Params that are not usable as standalone are not option so use them as scalar values.